controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -44,12 +60,26 @@ class ControllerInterface : public rclcpp_lifecycle::node_interfaces::LifecycleN
   virtual
   ~ControllerInterface() = default;
 
+  CONTROLLER_INTERFACE_PUBLIC
+  virtual
+  InterfaceConfiguration command_interface_configuration() const = 0;
+
+  CONTROLLER_INTERFACE_PUBLIC
+  virtual
+  InterfaceConfiguration state_interface_configuration() const = 0;
+
+  CONTROLLER_INTERFACE_PUBLIC
+  void assign_interfaces(
```

Comment:
```suggestion
  void claim_interfaces(
```
We've used the word `claim` for this throughout ros_control

Reply:
@bmagyar I was at first with you on this. But later, when I saw CM, I am actually with @Karsten1987 on this.
This function is called from CM, which is actually assigning interfaces... This is semantically more correct. The logic in this concept is somewhat different, so that CM is assigning interfaces and not that controller claims them from hardware.

Claiming is happening [few lines before](https://github.com/ros-controls/ros2_control/pull/236/files#diff-51a3ffe225a0370e8b9026ea1cb8f0cf34fd5bf6547a239ce8418f445e22a094R637).

Reply:
same here, claiming is not exposed in the controller interface anymore. The controller has no control over the controller manager neither the hardware.